selfdrive/camerad/test/check_exposure.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
```

Comment:
rename the file to `test_exposure.py` to match the convention used throughout the codebase